src/components/animated/_expanding.less

```diff
@@ -8,8 +8,18 @@
       height 450ms cubic-bezier(0.07, 0.62, 0.61, 0.3),
       opacity 450ms cubic-bezier(0.07, 0.62, 0.61, 0.3);
   }
+
   opacity: 1;
   border: 0.2rem solid #333;
+
+  [data-theme="dark"] &,
+  &.dark-mode {
+    &:before {
+      border: 0.2rem solid #fff;
```

Comment:
Instead of repeating this line, change the selector from `&:before` to `&, &:before`